CodeStarterCamp_Week1/Helper.swift

```diff
@@ -0,0 +1,54 @@
+//
+//  Helper.swift
+//  CodeStarterCamp_Week1
+//
+//  Created by QL Jeong on 2023/08/17.
+//
+
+import Foundation
+
+// 정해진 문자열을 줄바꿈 포함하여 반복하여 반환하며 마지막은 줄바꿈이 없음
+func repeatRowString(_ repeating: String, count: Int) -> String {
```

Comment:
[Swift API Design Guidelines](https://www.swift.org/documentation/api-design-guidelines/#promote-clear-usage) 문서를 참고하시면 아래와 같은 내용을 확인하실 수 있습니다!
Promote Clear Usage - **Omit needless words**

해당 내용에 따라 Helper 파일 전반적으로 함수 이름들을 수정해보시면 좋겠네요 🙂
```suggestion
func repeat(rowString: String, count: Int) -> String {
```